Dockerfile

```
@@ -2,11 +2,11 @@ FROM qmkfm/base_container
 MAINTAINER Zach White <skullydazed@gmail.com>
 
 RUN apt-get update && apt-get install --no-install-recommends -y \
-    redis-tools && rm -rf /var/lib/apt/lists/*
+    redis-tools clang && rm -rf /var/lib/apt/lists/*
```

We have clang in base_container now.

```suggestion
    redis-tools && rm -rf /var/lib/apt/lists/*
```